### ISSUE TITLE
Rebuilding types on dever server reloads which is slowing down productivity

### ISSUE THREAD

**Issue by gregg-cbs** [mancku/strapi-plugin-schemas-to-ts#51]
Hey Mancku, I have noticed that my dev server is rebuilding types every time it restarts. Its worth noting that I run strapi in debugger mode which could have something to do with it.

If i restart my debugger I see a full rebuild happens everytime and my terminal is being spammed by logs from this package - any thoughts?

![image](https://github.com/mancku/strapi-plugin-schemas-to-ts/assets/57114335/2e25bdd7-3643-4c4b-baf6-0bf5d33a15eb)

To be fair, even strapi does quite a lot of rebuilding on debugger restarts.

**Comment by mancku** [comment]
Hi @gregg-cbs yes, it's something I couldn't figure out (how to avoid the plugin from being fired every time) but if you scroll down after the "Converting schema..." sentences, you'll find that files are not being modified every time, just when there are changes. When a file is not being modified, you'll see an "is up to date" message

**Comment by gregg-cbs** [comment]
Yeah its probably a webpack level thing and i think you will be blocked by that when working from within strapi. Looks like their diffing is not so great either.
